docs/DeveloperGuide.md

```diff
@@ -176,6 +176,29 @@ The mark feature mechanism is implemented by having the Event class keep track o
 
 **Implementation design**: The feature is implemented with a seperate class 'Mark' instead of a 'isDone' boolean attribute. This ensures that there is a greater abstraction and allows for further development if we wish to further develop the application to keep track of other milestones.
 
+### Unmark feature
+
+The `unmark` feature allows for unmarking of the 'Mark' attribute in events should there be an accidental marking of an event.
+
+#### Implementation
+
+The `unmark` feature is implemented with the following considerations:
+
+- A valid `index` will be taken in as a parameter to facilitate unmarking of the correct event.
+- The unmark feature builds on the existing implementation of the 'Mark' class.
```

Comment:
Is 'build alongside' or equivalent better? 'builds on' to me sounds like there are dependencies involved.

Reply:
yup sounds good, amended 👍